baseDecorator.rb

```diff
@@ -0,0 +1,14 @@
+require_relative 'nameable'
+
+class BaseDecorator < Nameable
+  attr_accessor :nameable
+
+  def initialize(nameable)
+    super()
+    @nameable = nameable
+  end
+
+  def correct_name
+    puts @nameable
+  end
```

Comment:
- [ ] This method should return @nameable.correct_name, not put the nameable, kindly fix this so it returns the correct value